civihr_employee_portal/views/templates/views-view-table--Documents--block.tpl.php

```diff
@@ -34,20 +37,20 @@
   $statusesCount[0] ++;
 endforeach;
 ?>
-<div class="chr_table-w-filters chr_table-w-filters--documents row">
+<div id="tnadocuments" ng-controller="MainCtrl" class="chr_table-w-filters chr_table-w-filters--documents row">
```

Comment:
Yes, and it is the controller containing the `modalDocument(...)` function, which is responsible to display document modal.